src/software/thunderscope/replay/proto_player.py

```diff
@@ -130,6 +134,91 @@ def is_log_entry_corrupt(log_entry: bytes, version: int) -> bool:
         except Exception:
             return True
 
+    def build_chunk_index(self, folder_path: str) -> dict[str:float]:
```

Comment:
Consider using `os.PathLike` for `folder_path` instead of `str`

Reply:
Done